call-home/src/main.rs

```diff
@@ -0,0 +1,124 @@
+use std::{thread, time};
+pub mod collector;
+pub mod common;
+use clap::{App, Arg};
+use tokio;
+use sha256::digest;
+use crate::collector::http_client::HttpClient;
+use crate::collector::k8s_client::K8sClient;
+use crate::collector::report_models::{Pools, Replicas, Report, Volumes};
+use tracing::{info, Level, error,warn};
+use tracing_subscriber::FmtSubscriber;
+
+const PRODUCT: &str = common::constants::PRODUCT;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::TRACE)
+        .init();
+
+    let matches = App::new(clap::crate_description!())
```

Comment:
Why not use the derive api instead? It's much neater and avoids unwraping later.